src/main/webapp/script.js

```diff
@@ -132,3 +151,10 @@ function refreshCalendar() {
   calendarIframe.src = calendarIframe.src;
 }
 
+/**
+ * Toggles all tasks checkboxes.
+ */
+function toggleAll() {
+  const toggleStatus = $("#toggle-all").prop("checked");
+  $("input[type=checkbox], input[name=taskId]").prop("checked", toggleStatus);
```

Comment:
This selector would change all the checkboxes anywhere. So far this is okay, since there are no other checkboxes on the page. But imagine adding a new checkbox for controlling a new feature - it would be messed up as well.
Consider changing the selector to `$("#task-list input[type=checkbox]")`, that should do.